v4/export/prepare.go

```diff
@@ -8,6 +8,10 @@ import (
 )
 
 func adjustConfig(conf *Config) {
+	if !conf.EscapeBackslash {
+		globalEscape = &noBackslashEscape{}
+	}
```

Comment:
When I said "global config" I mean a config variable used by every procedure, not a literal global variable.